.circleci/config.yml

```diff
@@ -293,6 +302,30 @@ jobs:
           to: s3://${AWS_BUCKET}/widget-demo/production
           overwrite: true
 
+  tag_for_production:
+    executor: main-executor
+    working_directory: ~/react-widgets
+    steps:
+      - checkout
+      - run: git remote add upstream git@github.com:webex/react-widgets.git
+      - run: git fetch --all --tags --prune
+      - run:
+          name: 'Grab Version Number'
+          command: |
+            echo export VERSION_NUMBER=$(curl 'https://code.s4d.io/widget-demo/production/.version') >> $BASH_ENV
```

Comment:
Could you rename the variable to `PRODUCTION_VERSION_NUMBER` so that it is different from the `VERSION_NUMBER` used in the previous jobs.